DESCRIPTION

```diff
@@ -2,20 +2,19 @@ Package: greta.gp
 Type: Package
 Title: Gaussian Process Modelling in greta
 Version: 0.1.3
-Date: 2017-09-09
+Date: 2019-02-21
 Author: Nick Golding
```

Comment:
please add your self to the authorship!
Code example here: https://github.com/greta-dev/greta/blob/master/DESCRIPTION#L6-L21

Reply:
done. added as ctb and updated to use the suggested format